src/autocomplete.js

```diff
@@ -165,7 +164,7 @@ export default class Autocomplete extends Component {
       menuOpen: newState.menuOpen || false,
       query: newQuery,
       selected: null
-    })
+    }, this.props.confirmOnBlur ? this.props.onConfirm(newQuery) : null)
```

Comment:
@tvararu I wasn't quite sure how the `selected` property was being used so I opted to just send `newQuery`. During testing, `selected` was `-1` so `option[selected]` was not returning what I expected.